tailwind.config.js

```diff
@@ -35,12 +35,14 @@ module.exports = {
     },
     screens: {
       sm: "640px",
+      maxsm: { min: "100px", max: "767px" },
       md: "768px",
```

Comment:
Instead of `maxsm` you can override classes going bottom-up, for example, instead of:

```
maxsm:flex-col
```

you can do

```
flex-col md:flex-row
```

This way we don't need multiple variants of each breakpoint :)